config.sh

```diff
@@ -6,8 +6,12 @@ LOCAL_FT_COMMIT=5ad9b15
 
 # Test arguments
 NPROC=2
-PYTEST_ARGS="-ra --maxfail=1 --timeout=300 --durations=25 -n $NPROC"
+PYTEST_ARGS="-ra --maxfail=1000 --timeout=300 --durations=25 -n $NPROC"
 
+function pip_opts {
+    # Define extra pip arguments
```

Comment:
cryptography dropped 32 bit linux support. this encourages pip to use the latest wheel that satisfies the requirements.